.travis.yml

```diff
@@ -20,7 +20,6 @@ matrix:
       dist: precise
 
 before_script:
-  - composer self-update
```

Comment:
Any reason to not ask Travis to update composer?

Reply:
The Travis build will do this command by default.
You can see more details about the [job log](https://travis-ci.org/peter279k/cldr-to-gettext-plural-rules/jobs/363964909).